src/Drupal/Commands/field/FieldDefinitionCommands.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drush\Drupal\Commands\field;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Field\FieldTypePluginManagerInterface;
+use Drupal\Core\Field\FormatterPluginManager;
+use Drupal\Core\Field\WidgetPluginManager;
+use Drush\Attributes as CLI;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Defines field definition Drush commands.
+ */
+final class FieldDefinitionCommands extends DrushCommands {
+
+    public function __construct(
+        private readonly FieldTypePluginManagerInterface $typePluginManager,
+        private readonly WidgetPluginManager $widgetPluginManager,
+        private readonly FormatterPluginManager $formatterPluginManager,
+    ) {
+        parent::__construct();
+    }
+
+    #[CLI\Command(name: 'field:types')]
+    #[CLI\FieldLabels(
+        labels: [
+            'id' => 'ID',
+            'label' => 'Label',
+            'default_widget' => 'Default Widget',
+            'default_formatter' => 'Default Formatter',
+            'settings' => 'Settings',
+        ],
+    )]
+    #[CLI\Help(
+        description: 'Lists field types.',
+        synopsis: 'drush field:types [options]',
```

Comment:
Lets rely on Usages instead of this synopsis.

Reply:
Also all other core commands use Doxygen for description and synopsis which is a bit more readable than Help attribute in my opinion and @greg-1-anderson opinion. Its a taste thing so OK to leave as is.

Reply:
Replaces with Usage attribute.